ios/Runner/Info.plist

```diff
@@ -23,15 +25,17 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.12.9</string>
+	<string>0.12.10</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleURLTypes</key>
 	<array>
 		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Viewer</string>
 			<key>CFBundleURLSchemes</key>
 			<array>
-				<string>$(GOOGLE_SIGNIN_URL)</string>
+				<string>com.googleusercontent.apps.627353432544-5n2dacud5i80vt5t76f41b0q0bh2iruk</string>
```

Comment:
The property has to be changed to store the backup correctly but it is good that you paid attention to the file. I think it will be great to add it to gitignore.